src/components/rules/SubPages.tsx

```diff
@@ -0,0 +1,47 @@
+import { Badge } from '@/components/ui/Badge';
+import { Link } from '@/lib/locale/navigation';
+import Image from 'next/image';
+
+type SubPagesProps = {
+  className?: string;
+  id: number;
+  internal: boolean;
+  title: string;
+  photoUrl: string;
+};
+
+function SubPages({ className, id, internal, title, photoUrl }: SubPagesProps) {
+  return (
+    <Link
+      href={{
```

Comment:
When putting a link around something and it doesnt have text it needs an aria-label that explains what it does. I think, this may be wrong other places in the app too, maybe we should look into this